src/Shared/USConverters/CorrelationMatrixConverter.ts

```diff
@@ -0,0 +1,105 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Jedrzej Ruta
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import MorningstarConverter from '../MorningstarConverter';
+import type { XRayUSConverterOptions } from '../../XRayUS/XRayUSOptions';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class CorrelationMatrixConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: XRayUSConverterOptions
+    ) {
+        super(options);
+    }
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: XRayUSConverterOptions,
+        hasMultiple?: boolean
+    ): void {
+        const table = this.table,
+            columnSuffix = hasMultiple ? `_${options.json.PortfolioName}` : '',
+            correlationMatrix = options.json.Risks.CorrelationMatrix;
+
+        if (correlationMatrix.length) {
```

Comment:
That causes an error: 

<img width="2290" height="1040" alt="Image" src="https://github.com/user-attachments/assets/b3269c33-6c37-4b69-9688-6b076884d71b" />